sql/core/src/test/scala/org/apache/spark/sql/ExpressionsSchemaSuite.scala

```diff
@@ -152,7 +152,7 @@ class ExpressionsSchemaSuite extends QueryTest with SharedSparkSession {
 
     val outputSize = outputs.size
     val headerSize = header.size
-    val expectedOutputs: Seq[QueryOutput] = {
+    val (expectedMissExamples, expectedOutputs): (Array[String], Seq[QueryOutput]) = {
```

Comment:
No big deal, but the types can just be added to the two tuple elements, instead of declaring them separately as a type for the whole tuple

Reply:
First changed to `val (expectedMissingExamples: Array[String], expectedOutputs: Seq[QueryOutput])`, but I feel that the type declaration here is redundant, so changed to `val (expectedMissingExamples, expectedOutputs)`. Is this acceptable？